### PR TITLE
Add contribution links within context #1000 #1001

### PR DESCRIPTION
**Pull request by Tosin-Balogun**
## Description

This is a prototype based on improving the users access to our contribution process through the service manual website

We will use this pull request to review changes, updates et al
### Related issue

#1000 #1001 
## Checklist

- [x] Tested against the [NHS.UK frontend testing policy](https://github.com/nhsuk/nhsuk-frontend/blob/master/docs/contributing/testing.md) (Resolution, Browser & Accessibility)
- [x] Code follows the [NHS.UK frontend coding standards](https://github.com/nhsuk/nhsuk-frontend/blob/master/docs/contributing/coding-standards.md)
- [ ] Version number has been increased in `package.json` (using [SEMVER](https://semver.org/))
- [ ] CHANGELOG entry
- [ ] [Whats new page](https://service-manual.nhs.uk/whats-new) entry
- [ ] Page updated date

### CODE REVIEW

**Comment by Tosin-Balogun** [comment]
The conclusion was to see if we can repurpose the 'get in touch' partial to host the contextual links idea. We would need to restyle and rewrite it based on the evidence we have discovered

@davidhunter08 - The above is correct, yes?
@sarawilcox @nancyhalladay24 
Screenshot 2021-03-23 at 17 14 48

File: app/views/includes/_contact-panel.njk [hunk 1]
```diff
@@ -1,34 +1,24 @@
...
+<div class="nhsuk-action-link">
+  <a class="nhsuk-action-link__link" href="https://github.com/nhsuk/nhsuk-service-manual-backlog/issues/{{backlog_issue_id}}">
+    <svg class="nhsuk-icon nhsuk-icon__arrow-right-circle" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
+      <path d="M0 0h24v24H0z" fill="none"></path>
+      <path d="M12 2a10 10 0 0 0-9.95 9h11.64L9.74 7.05a1 1 0 0 1 1.41-1.41l5.66 5.65a1 1 0 0 1 0 1.42l-5.66 5.65a1 1 0 0 1-1.41 0 1 1 0 0 1 0-1.41L13.69 13H2.05A10 10 0 1 0 12 2z"></path>
+    </svg>
+    <span class="nhsuk-action-link__text">Share your research findings </span>
+  </a>
 </div>
+<p>If you have any questions, you can <a href="https://service-manual.nhs.uk/slack">reach out to our community forum on Slack</a> or <a href="mailto:service-manual@nhs.net">contact us via email</a>.
```
Comment (davidhunter08) [inline app/views/includes/_contact-panel.njk:22]: "via email" or "by email"?

Comment (Tosin-Balogun) [reply app/views/includes/_contact-panel.njk:22]: @sarawilcox - What do you reckon? Is the word 'via' not as accessible as 'by'?

File: app/views/includes/_contact-panel.njk [hunk 2]
```diff
@@ -1,34 +1,24 @@
...
-  {% else %}
-    <p>The service manual is a community effort. Anyone can help improve and grow it.</p>
-    {% if subSection === "Styles" or subSection === "Components" or subSection === "Patterns" %}
-    <p>If you spot a problem with the {{pageTitle}} guidance you can <a class="app-contact-panel__link" href="https://github.com/nhsuk/nhsuk-service-manual/blob/master/app/views/design-system/{{subSection|lower}}/{{pageTitle|lower|replace(" ", "-")|replace("'", "")}}.njk">propose a code change</a>.</p>
-    {% else %}
-    <p>If you spot a problem with the {{pageTitle}} page you can <a class="app-contact-panel__link" href="https://github.com/nhsuk/nhsuk-service-manual/blob/master/app/views/{{subSection|lower|replace(" ", "-")|replace("'", "")}}/{{pageTitle|lower|replace(" ", "-")|replace("'", "")}}.njk">propose a code change</a>.</p>
-    {% endif %}
-  {% endif %}
+<p> If so, please share your research findings and let us know how it has worked for you. This will help us improve it and help others.</p>
+<p>Before you start, you will need a <a href="https://github.com/join?source=login">GitHub account</a>, It's an open forum where we collect feedback</p>
```
Comment (davidhunter08) [inline app/views/includes/_contact-panel.njk:6]: small "i" needed on "it's an open forum..."

**Review by davidhunter08 — CHANGES_REQUESTED** [review]

**Review by sarawilcox — COMMENTED** [review]
Hi Tosin, could you please make this 2 sentences. It should read: 
<p>Before you start, you will need a <a href="https://github.com/join?source=login">GitHub account</a>. It's an open forum where we collect feedback.</p>

**Comment by Tosin-Balogun** [comment]
> Hi Tosin, could you please make this 2 sentences. It should read:
> 
> Before you start, you will need a [GitHub account](https://github.com/join?source=login). It's an open forum where we collect feedback.

Done ✅

**Review by sarawilcox — APPROVED** [review]
Thanks Tosin.

**Comment by Tosin-Balogun** [comment]
@DomBaker - this one

**Pull request merged**